buildpack/util.py

```diff
@@ -158,6 +158,10 @@ def get_buildpack_loglevel():
     return logging.INFO
 
 
+def is_buildpack_debug_logging_enabled():
```

Comment:
Is this utility function really necessary?
I see two usages which can be replaced with either a logic evaluation or something like `str()`.

Reply:
this has been removed